### DISCUSSION TITLE
using tile Identifier in eodag CLI?

### DISCUSSION THREAD

**Discussion by frazucca** [CS-SI/eodag#1394] [General]
Hi at all,
I wonder about the possibility to use in _eodag search_ in CLI something related to S2 tile (in copernicus dataspace).

TIA
Best,
franz

**Comment by sbrunato** [comment]
Hello @frazucca ,

you can perform this search using
```sh
eodag search -p S2_MSI_L1C -q "provider=cop_dataspace&tileIdentifier=31TCJ"
```

**Comment by frazucca** [comment]
Ciao Sylvain
thank you!!
fz

Il giorno gio 14 nov 2024 alle ore 15:05 Sylvain Brunato <
***@***.***> ha scritto:
